recipe/meta.yaml

```diff
@@ -10,40 +10,49 @@ package:
 source:
   fn: cvxopt-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/c/cvxopt/cvxopt-{{ version }}.tar.gz
-  sha256: 8f157e7397158812cabd340b68546f1baa55a486ed0aad8bc26877593dc2983d
+  sha256: 3296c9d49b7dcb894b20db5d7d1c1a443912b4d82358e03f836575e8398e0d60
 
 build:
-  skip: true  # [win]
-  number: 202
+  skip: true  # [win32 or (win and py<35)]
+  number: 200
   features:
     - blas_{{ variant }}
+    - vc14    # [win and py>=35]
 
 requirements:
   build:
     - python
     - setuptools
     - blas 1.1 {{ variant }}
     - openblas 0.2.20|0.2.20.*
-    - gsl
-    - fftw
+    - gsl     # [not win]
+    - fftw    # [not win]
```

Comment:
FWIW we do have `gsl` and `fftw` for Windows as well.

Reply:
I tried to add gsl and fftw, but it may require patching some header files. MSVC14 handles complex numbers in a non-standard way... (`_Dcomplex` instead of `complex double`, etc.)

Reply:
I see. @jschueller and @dougalsutherland have worked on Windows builds of `fftw` and `gsl`. They may be able to advise (if they have time).